app/views/cookie_preferences/new.html.erb

```diff
@@ -0,0 +1,75 @@
+<%= content_for :page_title, "Cookies" %>
```

Comment:
should the page title and url not match?

Reply:
I don't think the page title needs to match the url but I do believe it should match the page title which is why I went with `Cookies`